openpype/hosts/maya/hooks/pre_auto_load_plugins.py

```diff
@@ -0,0 +1,15 @@
+from openpype.lib import PreLaunchHook
+
+
+class PreAutoLoadPlugins(PreLaunchHook):
+    """Define -noAutoloadPlugins command flag."""
+
+    # Execute before workfile argument.
+    order = 0
+    app_groups = ["maya"]
+
+    def execute(self):
+        maya_settings = self.data["project_settings"]["maya"]
+        if maya_settings["explicit_plugins_loading"]["enabled"]:
+            self.log.debug("Explicit plugins loading.")
+            self.launch_context.launch_args.append("-noAutoloadPlugins")
```

Comment:
Is there no way that maybe from this plug-in, if we order it earlier than the "add last workfile arg" hook that we can just pass something along the launch context's or alike so that the other hook knows "FORCE POST LAUNCH" hook.

E.g. we could maybe set something like `force_no_workfile_arg=True`. We could then do the same from a prelaunch hook for the post initialization option so that it doesn't need to be specifically checked for the maya settings in the add last workfile args prelaunch hook.

I think it could clean up the code nicely.

Reply:
Lets continue this conversation in the other comment.